data/strings/sound.txt

```diff
@@ -478,6 +678,7 @@
     de = Sie sind angekommen.
     el = Έχετε φτάσει.
     es = Ha llegado a su destino.
+    es-MX = Ha llegado a su destino.
```

Comment:
Do es-MX and pt-BR correctly take es/pt translation if it's equal and missing?

Reply:
I haven't tested this, `es-MX` appeared on my radar during development and I wanted to stub it out / handle it. AFAIK es-MX is currently equal to es, however pt-BR is definitely not equal to pt in all cases. I can remove either/or if you desire, I don't intend a lot of changes there.

Reply:
Please don't change PT and PT-BR, I've translated and revised them. Yes, there are some differences.

Reply:
@matheusgomesms can you review the PR as it currently looks and tell me if the current changes are correct? https://github.com/organicmaps/organicmaps/pull/3130/files/00cb82086690e58e6edde753ed166cbe5675f537#diff-631491c869fee64c6cc86150edcfb7f52b23d306aee68d05816f8a141727e022

The only additional change I plan to make tonight is the change you requested on September 8:

```diff
   [take_exit_number]
     en = Take exit
     ar = خروج إلى
     be = З'езд на
@@ -609,8 +569,8 @@
     nb = Ta avkjøringen inn på
     nl = Verlaat naar
     pl = Wybierz zjazd
-    pt = Saia para
-    pt-BR = Pegue a saída para
+    pt = Apanhe a saída
+    pt-BR = Pegue a saída
```

Reply:
Great, thanks! I got confused with many comments :P

I checked the Portuguese community, and it seems that the best approach here is (I guess already implemented on lines 572-573):

pt = Apanhe a saída
pt-BR = Pegue a saída